src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,11 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String text1 = text.toLowerCase().replaceAll("[^\\w\\d]", "");
+        StringBuilder palindrome = new StringBuilder(text1).reverse();
+        String text2 = palindrome.toString();
+        return text1.equals(text2);
```

Comment:
круто, але це можна трішки спростити:
```suggestion
        String replacedText = text.toLowerCase().replaceAll("[^\\w\\d]", "");
        StringBuilder palindrome = new StringBuilder(text1).reverse();
        return replacedText.equals(palindrome.toString());
```